### PR TITLE
AB#72018: Adding shared ymls

### PR DESCRIPTION
**Pull request by nicholaswilson100**
These are 2 resuable ADO tasks/actions used in ADO pipelines.

They are used, for example, here: 

https://github.com/oxctl/signup/pull/181

line 121:     - template: test_deployment.yml@pipeline-templates

I haven't fully parameterised them yet because I'm not sure if we're progressing this work

### CODE REVIEW

File: react-deploy-to-s3-action.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Sync a built React application to an AWS S3 repository.
+
+parameters:
+  - name: bucket
+    type: string
+
+steps:
+  - task: AWSShellScript@1
+    inputs:
+      awsCredentials: 'OUIT PROD'
```
Comment (buckett) [inline react-deploy-to-s3-action.yml]: Should we pass this in as a parameter so that we can use it for both prod and dev?

Comment (buckett) [reply react-deploy-to-s3-action.yml]: @nicholaswilson100 What did you think to this?

Comment (nicholaswilson100) [reply react-deploy-to-s3-action.yml]: Yeah you're right. - I didn't do it because the last thing I heard from Adam was that we weren't doing pipleines

Comment (nicholaswilson100) [reply react-deploy-to-s3-action.yml]: Yeah let me change this  - should do it properly I suppose even if it's not going to be used

**Review by buckett — APPROVED** [review]
Looks good to me.

**Comment by nicholaswilson100** [comment]
Thanks for comments

**Pull request merged**